packages/loader/container-loader/src/containerContext.ts

```diff
@@ -15,7 +15,7 @@ import {
 	IFluidCodeDetails,
 	IBatchMessage,
 } from "@fluidframework/container-definitions";
-import { FluidObject } from "@fluidframework/core-interfaces";
+import { FluidObject, ISignalEnvelope } from "@fluidframework/core-interfaces";
```

Comment:
Both of these are type only imports.
(There is a lint rule, but it hasn't been broadly deployed yet.)